components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/ApplicationAuthenticator.java

```diff
@@ -85,6 +85,18 @@ public AuthenticatorFlowStatus process(HttpServletRequest request,
      */
     public String getClaimDialectURI();
 
+    /**
+     * Get the claim dialect URI if this authenticator receives claims in a standard dialect
+     * and needs to be mapped to the Carbon dialect http://wso2.org/claims
```

Comment:
Java Doc description should end with a period (.)

Reply:
fixed.